tests/factories/cht/reports/pregnancy.js

```diff
@@ -15,6 +17,9 @@ const defaultSubmitter = {
   }
 };
 
+const nextANCVisit = moment().add(2, 'day');
```

Comment:
This is the date when the CI runs and adds two days, right? 

There's a possibility that `nextANCVisit` is next month. If the current date is 30/12/2024, then `nextANCVisit` is 01/01/2025. 
Will that make any of the e2e tests fail? We want to avoid blocking devs because all the PR's CI fails at the end of the month.

Reply:
No it won't fail. I am not checking the value of the `nextANCVisit` in the `contact-details.wdio-spec.js`.  The value was added so that calling [getPregnancyCardInfo](https://github.com/medic/cht-core/blob/c2393c7ec0beda3802e856a825fb7521e759bee3/tests/page-objects/default/contacts/contacts.wdio.page.js#L314) in the test does not fail